### ISSUE TITLE
Map index problem

### ISSUE THREAD

**Issue by xuhao-anhe** [open-mmlab/mmdetection#7766]
Hello, I'm using point_ During the rend experiment, my map changes are very jittery and often reach the peak in the middle of training, but the highest value only appears a few times, and then begins to decline and rise. Is this normal

**Comment by RangiLyu** [comment]
Maybe your model has started to overfit. Try to tune the hyperparameters. Also, you can try to decay the learning rate when mAP stops increasing.
